### ISSUE TITLE
Pusillin and the Elder Azj'Torin

### ISSUE THREAD

**Issue by jarvik76** [WoWManiaUK/Redemption#3072]
Quest ID 7441 Pusillin and the Elder Azj'Torin
When you speak to Pusillin to start the chase, instead of running around the balcony, he runs straight across the "sky" in the courtyard towards the opposite hallway, as if it was solid ground.  This means because of the way escort quests NPC despawn, quest cannot be completed because he despawns before he can be found in the end balcony at the end of the hallway.
Occasionally he will run again across the sky, but diagonally across towards the opposite corner.  Once he DID run up the balcony in the expected direction, but then cut the corner at the end and fell through to the courtyard below.
  
This means also you're trapped because the door at the end of the chase requires the key that he drops to open it. Only way out is to HS or die, or run all the way back to the start.
![wowscrnshot_022219_123214](https://user-images.githubusercontent.com/46089380/53243020-7bb98800-369e-11e9-8701-ffe61c32ba0f.jpg)
![wowscrnshot_022219_123208](https://user-images.githubusercontent.com/46089380/53243021-7bb98800-369e-11e9-87c6-a6132afebdda.jpg)
![wowscrnshot_022219_123203](https://user-images.githubusercontent.com/46089380/53243023-7c521e80-369e-11e9-8bbc-7731fbfd4304.jpg)
![wowscrnshot_022219_123159](https://user-images.githubusercontent.com/46089380/53243024-7c521e80-369e-11e9-8a3e-4888e9bdfe48.jpg)

**Comment by wow-mania** [comment]
You need to stay close to it

**Comment by jarvik76** [comment]
yes.  you can't... because he runs straight across the sky in the open courtyard.  As if it was solid ground.   So you can't follow him.  I appreciate it's difficult to see from this screenshot.  
The third one down is most obvious.  He's supposed to run around the stone balcony

UPDATE: I just tried again, and instead of starting the chase, he stood there and fought me.

**Comment by jarvik76** [comment]
Perhaps this shot makes more sense.
I am stood where he starts.  He's supposed to run to the top right, along the balcony.  Instead, he goes directly left, running across the tops of the trees as if he can float.
![wowscrnshot_022219_124357](https://user-images.githubusercontent.com/46089380/53243386-950f0400-369f-11e9-9538-70ba1e1e15e8.jpg)

**Comment by wow-mania** [comment]
If you let it open on you it will change its path and cut through

**Comment by wow-mania** [comment]
OK I forgot to mention, every time it stops running you need to talk to him again and make it continue or it will stop and go away. This is Blizzlike.

**Comment by WhitephoenixTheBunny** [comment]
I just have tried the quest several times. Sometimes, the NPC will take a wrong path and go in diagonal and float in the air. It seems beeing a wrong respawn, nothing that a ticket would solve, you can ask a GM to reset the NPC, it will start on correct path after 2 or 3 resets, or go outside and reset instance. On my tries, he will take wrong path on 2/10

![image](https://user-images.githubusercontent.com/31921098/53244322-70685b80-36a2-11e9-8f7b-9f9b929e269c.png)

**Comment by jarvik76** [comment]
I don't understand why this was closed
It is not that you can't talk to him.  It's that he runs straight across the open sky where the player cannot follow.

**Comment by Echinaceax** [comment]
I believe it was closed coz the fix is just to make a ticket "in game" and ask GM to rest npc  ID7441

> nothing that a ticket would solve, you can ask a GM to reset the NPC, it will start on correct path after 2 or 3 resets

**Comment by Dranah** [comment]
Just to be clear, this has been broken as long as mania exists. Reported it once before https://github.com/WoWManiaUK/Redemption/issues/1506

**What should happen:** 
Pussilin should run a part of his path each time you speak to him.
He moves (along the path), you follow, whenever he stops, you speak to him again. Each time you speak to him, he says something stupid in a rhyme, and runs the next mile. After the last segment, he reaches his destinated spot in the room of satyrs, up the ramp and becomes attackable, you slay him and gg.

**What happens instead:**
Pussilin runs the first section properly, however when he stops, he despawns when you speak to him (or after a certain time , if you dont) and instantly teleports back to where he started at the entrance. You have to walk back and talk to him again.
The second time, he'll run the first+second section at once, cutting off the road through the walls/air. Means you can't follow him. You can go around and slowly catch up but he'll despawn and be back at the entrance by the time you reach him.
He repeats it a 3rd and a 4th time, each time running further than the previous (through the walls/air) and each time teleporting back to the entrance afterwards.
The last time you speak to him, he simply becomes attackable and you slay him.

TL;DR his script has one "minor" mistake, which is that he keeps teleporting back to the entrance after each running section. This makes the party run back and forth like an idiot. Doable? Yes. Broken as hell? Yes. Takes 5x more time and is not blizzlike at all.

It may be a tweaking issue (since is doable), but an issue nonetheless.

**Comment by Kitzunu** [comment]
Fixed.
He runns the correct Path, does not fly in the air.
Continues to run when you talk with him.
Let's you kill him at the end, to get the book
